_posts/2019-04-25-is-cv-the-best-way-to-hire-a-software-developer.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Is a CV the best way to hire a software developer?
+date: 2019-04-25 00:00:00
+featured_image: "/images/cv-resume/featured.jpg"
+excerpt: I've been on both sides of the table as an applicant and as an interviewer. In both cases I cannot help but feel I'm engaging in a bullshit process that supposedly is meant to either exemplify ones experience or vet them. Surely there must be a simpler way.
+---
+
+![Photo by Clem Onojeghuo on Unsplash](/images/cv-resume/featured.jpg)
+
+I really hate doing things I know are borderline bullshit just because they are conventional wisdom. CVs are definitely one of them. First of all, biases always find their way around a CV. Age, sex and name are just some of the information we now reduct or obfuscate in order to remove certain biases. I would argue we need to deduct the whole CV in order to be somewhat impartial and effective. But then, you might ask, how do we vet people if not through a CV? How would you know about their background, skills and experience if you don't read their CV?
```

Comment:
`reduct` is a term that has a heavy algebraic context.

```suggestion
I really hate doing things I know are borderline bullshit just because they are conventional wisdom. CVs are definitely one of them. First of all, biases always find their way around a CV. Age, sex and name are just some of the information we now obfuscate in order to remove certain biases. I would argue we need to deduct the whole CV in order to be somewhat impartial and effective. But then, you might ask, how do we vet people if not through a CV? How would you know about their background, skills and experience if you don't read their CV?
```